app/src/main/java/com/example/imagefilter/ui/mainScreen/component/TopAppBar.kt

```diff
@@ -0,0 +1,23 @@
+package com.example.imagefilter.ui.mainScreen.component
+
+import androidx.compose.material.Text
+import androidx.compose.material.TopAppBar
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.tooling.preview.Preview
+import com.example.imagefilter.R
+
+@Composable
+fun AppToolBar() {
```

Comment:
Nit - Generally the first component of a file has the same name as of the file itself (i.e. in this case we should either rename the composable to `TopAppBar` or the file itself to `AppToolBar`.